[2025-04-04T12:48:04.203+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T12:47:45.216378+00:00 [queued]>
[2025-04-04T12:48:04.211+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T12:47:45.216378+00:00 [queued]>
[2025-04-04T12:48:04.212+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-04T12:48:04.225+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-04 12:47:45.216378+00:00
[2025-04-04T12:48:04.230+0000] {standard_task_runner.py:60} INFO - Started process 2836 to run task
[2025-04-04T12:48:04.233+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'manual__2025-04-04T12:47:45.216378+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmprych8cwj']
[2025-04-04T12:48:04.235+0000] {standard_task_runner.py:88} INFO - Job 174: Subtask Write_data_in_kafka
[2025-04-04T12:48:04.275+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T12:47:45.216378+00:00 [running]> on host 7fc40170f80d
[2025-04-04T12:48:04.350+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T12:47:45.216378+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T12:47:45.216378+00:00'
[2025-04-04T12:48:05.362+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-04T12:48:05.374+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Weather_Proj, task_id=Write_data_in_kafka, execution_date=20250404T124745, start_date=20250404T124804, end_date=20250404T124805
[2025-04-04T12:48:05.410+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-04T12:48:05.444+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
